### PR TITLE
"Portal Fleshlight & Underwear" overhaul

### PR DESCRIPTION
**Pull request by JerryWester**
# About The Pull Request
Changes a number of things with the portal fleshlight and underwear. See changelog for changes.

## Why It's Good For The Game
More potential interactions with these items.

## A Port?
No

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
fix: using unlinked fleshlight with unlinked underwear while equipped. Formerly showed up as if underwear wasn't equipped
tweak: portal fleshlight description
tweak: added orifices to `handle_post_sex`'s. Dunno if this interferes with anonymity
add: interactions with portal fleshlight
add: two new modes for portal panties (penis and mouth)
add: sprite work for penis (credit goes to @Moltovz from the Hyperstation 13 discord) and mouth mode
add: three modes for fleshlight when targetting groin (penis, vagina, and anus)
/:cl:

### CODE REVIEW

**Comment by JerryWester** [comment]
I should also probably consider using `CUM_TARGET_` in place of `ORGAN_SLOT_` in a few instances

File: modular_sand/code/game/objects/items/fleshlight.dm [hunk 1]
```diff
@@ -137,22 +169,40 @@
...
+			if("vagina")
+				V = portal_target.getorganslot(ORGAN_SLOT_VAGINA)
+			if("anus")
+				V = portal_target.getorganslot(ORGAN_SLOT_BUTT)
+			if("penis")
+				V = portal_target.getorganslot(ORGAN_SLOT_PENIS)
+		/*
 		if(portalunderwear.targetting == "vagina")
-			V = portal_target.getorganslot(ORGAN_SLOT_VAGINA)
+			V = portal_target.getorganslot(ORGAN_SLOT_VAGINA) // Why doesn't this have an equivalent for the anus?
```
Comment (NullFag) [inline modular_sand/code/game/objects/items/fleshlight.dm:192]: Because there's no anus organ. ORGAN_SLOT_BUTT contains the asscheeks.

Comment (JerryWester) [reply modular_sand/code/game/objects/items/fleshlight.dm]: Gotcha, thank you for the heads up

**Comment by JerryWester** [comment]
For right now, I'm satisfied with the changes made. Further improvements are likely to be done in the future, but for now, I feel like this is enough.

**Comment by Moltovz** [comment]
Damn, Y'all gonna ban me from the server too or no?

**Comment by BongaTheProto** [comment]
> Damn, Y'all gonna **unban** me from the server too or no?

You still broke our rules. Come into the discord and make an appeal if you desire to be unbanned

**Comment by JerryWester** [comment]
> Damn, Y'all gonna **unban** me from the server too or no?

I do apologise if I've stepped on any toes, that wasn't my intention. I was informed by another user that you were okay with the sprites being used. If you'd like, I can try to backport it to Hyper, so it exists there as well, or I can find someone else to do the sprite work if you don't want it used at all.

**Comment by Moltovz** [comment]
> > Damn, Y'all gonna **unban** me from the server too or no?
> 
> You still broke our rules. Come into the discord and make an appeal if you desire to be unbanned

So I actually got banned by saviorhorse because we have a little history. I'm not interested in being unbanned, though.

**Comment by Moltovz** [comment]
> > Damn, Y'all gonna **unban** me from the server too or no?
> 
> I do apologise if I've stepped on any toes, that wasn't my intention. I was informed by another user that you were okay with the sprites being used. If you'd like, I can try to backport it to Hyper, so it exists there as well, or I can find someone else to do the sprite work if you don't want it used at all.

You didn't. This is hilarious.

**Comment by NullFag** [comment]
> > Damn, Y'all gonna **unban** me from the server too or no?
> 
> You still broke our rules. Come into the discord and make an appeal if you desire to be unbanned

Or just evade like I do.

**Comment by remis12** [comment]
okay sir but errrr 🤓 🤓 where's the sound design errr you know 🤓 🤓 the sounding?? 🤓 🤓 🤓

**Pull request merged**